draft-ietf-quic-manageability.md

```diff
@@ -809,20 +816,161 @@ off the connection ID may therefore cause undetectable and unrecoverable loss
 of state in the middle of a connection. Use of connection ID specifically
 discouraged for NAT applications.
 
-## Passive Network Performance Measurement and Troubleshooting
+## Network Address Translation (NAT)
 
-Limited RTT measurement is possible by passive observation of QUIC traffic;
-see {{sec-rtt}}. No passive measurement of loss is possible with the present
-wire image. Extremely limited observation of upstream congestion may be
-possible via the observation of CE markings on ECN-enabled QUIC traffic.
+This section uses the colloquial term Network Address
+Translation (NAT) to mean NAPT (section 2.2 of
+{{?RFC3022}}), which overloads several IP addresses to one IP address or to an
+IP address pool, as commonly deployed in carrier-grade NATs or residential NATs.
+
+{{?RFC4787}} contains some guidance on building NATs to interact constructively
+with a wide range of applications. This section extends the discussion to QUIC.
+
+In summary, the advise is simple: devices that perform NAT should use the
+QUIC connection ID to map their internal state but only use the 4-tuple of IP
+addresses and ports as they would do for other UDP traffic or other transports
+as well.
+
+As already recognised in section {#sec-stateful} the QUIC connection ID is not
+guaranteed to be constant over the life-time of a connection and a change of
+connection ID would therefore lead to loss of state or creation of unnecessary
+new state. Both should be avoided and indicate that a NAT device does not need
+to be QUIC-aware.
+
+Section  {#sec-stateful} also provides further recommendations on time-outs
+for stateful network functions such as NATs. For QUIC it is important that time-outs
+are not extremely short, however, the general recommendation as provided in
+{{RFC4787}} for UDP is also appropriate for QUIC and as such again not NAT
+function are not required to be QUIC-aware.
```

Comment:
This is getting quite repetitive.  Maybe you can just list things that don't work:

* Tracking connections based on connection ID (connection IDs change).
* Changing NAT timeouts based on detecting QUIC (QUIC doesn't care).
* Rewriting addresses to ensure stability (requires endpoint cooperation; enables amplification attacks).